tutorials/common/index-website.pug

```diff
@@ -32,8 +32,8 @@ mixin codeblock(format, step, parentstep, initial, stepname, target)
       .codeblock_step #{attributes.stepname}
       .codeblock_active Active
       .codeblock_inactive Inactive
-      button.codeblock_run Run
-      button.codeblock_reset Reset
+      button.codeblock_run.btn.btn-primary.btn-sm Run
+      button.codeblock_reset.btn.btn-link.btn-sm Reset
```

Comment:
We should add these classes to the `tutorials/common/index.pug` file as well, since I generate thumbnails from that and it will then look the same).

Reply:
Oh, when you generate, is it the exact same page as local tutorials? If so, I think I'll need to update the overwritten bootstrap styles to match the website first.

Reply:
I use xvfb and Chrome with the local tutorials to generate the thumbnails (that way it doesn't have the framing elements as part of the tutorial).  To my eye, the button style change is the only significant difference, so it would be sufficient.

Reply:
I should say, I don't think the font difference is that significant for the thumbnails.